model/publishing/event/RemoteDeliveryCreatedEvent.php

```diff
@@ -41,12 +43,21 @@ class RemoteDeliveryCreatedEvent extends AbstractDeliveryEvent implements Webhoo
     /** @var string|null */
     private $alias;
 
-    public function __construct(string $deliveryUri, string $testUri, string $remoteDeliveryUri, string $alias = null)
-    {
+    /** @var string|null */
+    private $webhookId;
```

Comment:
```suggestion
    private ?string $webhookId;
```

Reply:
I expect it to be backported to PHP7.4 versions